core/templates/dev/head/components/response_header_directive.html

```diff
@@ -49,6 +56,7 @@
     <span ng-if="isActive()">&nbsp;</span>
   </div>
 
+  <br class="break-in-mobile" />
```

Comment:
Just do `<br class="break-in-mobile">` -- br is a self-closing tag.

Reply:
Done